src/docs/rules.mdx

```diff
@@ -92,7 +92,7 @@ When speaking in either the global chat or the donator chat channel you should s
 
 ---
 
-## 3. Griefing
+## 3. Detrimental Inconveniences

```

Comment:
```suggestion
## 3. Griefing
```

Reply:
I think that "griefing" applies perfectly well to anything that could ruin someone's gameplay experience, this is an unnecessary change